### ISSUE TITLE
Can't ovveride to hide individual series from tooltip

### ISSUE THREAD

**Issue by KhoiDinh** [grafana/grafana#31375]
**What happened**:

I am trying to Use an override to hide individual series from the tooltip. However in the, "Overrides" tab for "TimeSeries" panel, there is no override property that I can choose to "hide individual series from the tooltip".

On the grafana doc website, for tooltip mode section's node, it says there is a way to do it but I dont see in the actual grafana tool: https://grafana.com/docs/grafana/latest/panels/visualizations/time-series/

**What you expected to happen**:

I expected to see an override property that lets me hide individual series from the tool tip.

**How to reproduce it (as minimally and precisely as possible)**:

Grafana -> add panel -> Panel (time series) -> overrides tab -> + Add override property 

- has no property to hide individual series from tool tip.

**Anything else we need to know?**:

**Environment**:
- Grafana version: 7.4.2
- Data source type & version:
- OS Grafana is installed on:
- User OS & Browser: Windows, Google Chrome Version 88.0.4324.182
- Grafana plugins:
- Others:

**Comment by ying-jeanne** [comment]
Hi @KhoiDinh I tried the steps locally, it seems working fine. You could find the tooltip mode setting under panel -> display -> tooltip mode. The ticket would be closed for now.

**Comment by skolarianer** [comment]
I can't find this override either. Yes, there is this tooltip mode setting but that's not what I'm looking for. I'm missing the "Hide in tooltip" series override from the now deprecated Graph visualization. The same applies for the legend override.

**Comment by dprokop** [comment]
@KhoiDinh @skolarianer this override is not available directly via the Overrides until you hide the series by clicking it in the legend. When you [toggle a series via the legend](https://grafana.com/docs/grafana/latest/panels/visualizations/panel-legend/#toggle-series) a new, so called "system" override will be created:

![image](https://user-images.githubusercontent.com/2376619/117766745-55a54180-b230-11eb-89d2-4f66c09cc883.png)

Now, you are able to toggle the series visibility in the legend and tooltip. And, also, bring back the series to be visible in the viz while hidden in the tooltip.

**Comment by skolarianer** [comment]
Thanks @dprokop, now I got it! I could swear I tried that before but didn't recognize the new override. I'd suggest to add the link you mentioned to this note of the timeseries doc:
![image](https://user-images.githubusercontent.com/6286711/117859109-e448ab80-b28e-11eb-909f-8a586d4271ac.png)


May I ask why this kind of override is not directly available? I mean ... I can add these system overrides almost like the other overrides via JSON:

```JSON
"overrides": [
  {
    "__systemRef": "hideSeriesFrom",
    "matcher": {
      "id": "byFrameRefID",
      "options": "XYZ"
    },
    "properties": [
      {
        "id": "custom.hideFrom",
        "value": {
          "graph": false,
          "legend": true,
          "tooltip": true
        }
      }
    ]
  }
]
```

Or is that not how it's supposed to be? Do I break something doing it this way? It's working fine for me so far.
